pages/blog/[id].tsx

```diff
@@ -9,11 +9,14 @@ import { useEffect, useState } from "react";
 const BlogInnerPage = () => {
   let { query: { id } } = useRouter();
   const [ post, setPost ] = useState(null);
+  const [ tags, setTags ] = useState(null);
 
   useEffect(() => {
     const fetchData = async () => {
       const { data: post } = await httpClient.get(`http://localhost/hydralab/wp-json/wp/v2/posts/${id}`);
+      const { data: tags } = await httpClient.get(`http://localhost/hydralab/wp-json/wp/v2/tags?post=${id}`);
```

Comment:
```suggestion
      const { data: tags } = await httpClient.get(`http://localhost/hydralab/wp-json/wp/v2/tags?post=${id}`);
```

this localhost should be replaced with localhost:PORT after latest Karen changes, and should be placed in environment variable